src/components/client/StatsPage.tsx

```diff
@@ -164,7 +168,7 @@ const StatsPage = () => {
                         >
                           {isFailedToFetchData
```

Comment:
Maybe we rename showData to a component that accepts <StatInfo ...{props} /> then we use composition api to render the outcome.